src/nft/components/profile/view/ProfilePage.tsx

```diff
@@ -152,7 +152,7 @@ export const ProfilePage = () => {
     if (ownerCollections?.length && collectionStats?.length) {
       const ownerCollectionsCopy = [...ownerCollections]
       for (const collection of ownerCollectionsCopy) {
-        const floorPrice = collectionStats.find((stat) => stat.address === collection.address)?.floorPrice
+        const floorPrice = collectionStats.find((stat) => stat.address === collection.address)?.stats?.floor_price //TODO update when changing walletStats endpoint to gql
```

Comment:
is this todo still relevant?

Reply:
Yes, this is related to a different endpoint being migrated https://github.com/Uniswap/interface/pull/5053